docker/fetch-data/Dockerfile

```diff
@@ -0,0 +1,26 @@
+
+FROM demisto/python3:3.8.3.9324
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git  g++ \
+  && apk add --no-cache python3-dev libstdc++  \
```

Comment:
Try moving `python3-dev` to the line above. It should be part of the `.build-dependencies`

Reply:
Thanks missed it, done